src/makeRobotAccountant.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+  let countAfterThree = 0;
```

Comment:
Do you really need this variable? Just use counter instead
![image](https://github.com/mate-academy/js_sum-with-closure/assets/96409563/4cec6727-9435-4cd9-8cd2-ffded7ac71ac)